src/lib/mina_base/signed_command_memo.ml

```diff
@@ -148,7 +148,7 @@ type raw = Digest of string | Bytes of string
 
 let to_raw_exn memo =
   let tag = tag memo in
-  if Char.equal tag digest_tag then Digest (to_string memo)
+  if Char.equal tag digest_tag then Digest (to_base58_check memo)
```

Comment:
Maybe comment this is still raw bytes as otherwise it may be misleading to the reader what format memo has.

Reply:
Ah - very nice - the separate interface files always catches me out!